### ISSUE TITLE
[Bug] shadowTrigger fails with options parameter

### ISSUE THREAD

**Issue by mandric** [abramenal/cypress-shadow-dom#22]
Maybe I am using the API incorrectly?  I know this example might not be the best one, but was curious why this test fails? 

```diff
diff --git a/example/cypress/integration/todolist.spec.js b/example/cypress/integration/todolist.spec.js
index fd89ff7..d189c9b 100644
--- a/example/cypress/integration/todolist.spec.js
+++ b/example/cypress/integration/todolist.spec.js
@@ -66,4 +66,10 @@ describe('Todo List', () => {
       .shadowFind('label')
       .shadowContains('async item');
   });
+
+  it('trigger allows options parameter', () => {
+    cy.shadowGet('todo-list-item', { timeout: 6000 })
+      .shadowFind('input')
+      .shadowTrigger('click', { timeout: 1000 });
+  });
 });
```
<img width="550" alt="Screen Shot 2019-11-25 at 1 05 56 PM" src="https://user-images.githubusercontent.com/25641/69569989-7b7b4f80-0f84-11ea-8b2a-519abf68667a.png">

**Comment by abramenal** [comment]
Hi @mandric
Thanks for submitting the issue!
Which version you’re using? I thought I’ve fixed that in the latest one.

Waiting for your reply

**Comment by mandric** [comment]
Oh sorry forgot to include that, using latest!
```
milan in ~/dev/scratch/cypress-shadow-dom/example (master *=)
$ npm ls cypress-shadow-dom
cypress-shadow-dom-example@1.0.0 /Users/milan/dev/scratch/cypress-shadow-dom/example
└── cypress-shadow-dom@1.2.0 
```

**Comment by mandric** [comment]
The test I wrote doesn't really make sense since the timeout is typically needed in the async loaded element, but I based on the docs (and probably api was designed for this support) it should still accept the parameter anyway?

**Comment by mandric** [comment]
Not sure if you agree or if it's even possible, but once you have an element, whether in Shadow DOM or not, it should conform to the same conditions Cypress has to define "actionability".  i.e. https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Actionability

So the API would looks something like this and then we can leverage all the work Cypress has done in the non shadow world?  So we can use the Cypress API within a shadow tree.  Otherwise not sure what is a viable solution.

```
  it('allows Cypress element API inside the shadowDom', () => {
    cy.shadowGet('todo-list-item', { timeout: 6000 })
      .get('input')
      .click();
  });
```

There is also a `within` call once you have an element, i.e. https://docs.cypress.io/api/commands/get.html#Get-in-within

I could see an API like this:

```
  it('allows Cypress element API inside the shadowDom', () => {
    cy.shadowGet('todo-list-item', { visible: true, timeout: 10000 }).within(() => {
        cy.get('input').click(); // cypress api available here inside shadowRoot
    });
  });
```

**Comment by abramenal** [comment]
@mandric you're right.
It seems that I've just added wrong parameter to docs, however it is not supported. Setting timeout is availalble for query commands, not for action ones – exactly like you stated above.
https://github.com/abramenal/cypress-shadow-dom/blob/master/index.d.ts#L8
I've updated README.md to reflect that as well.

As for `within` command, right now it is only supported in chaining way I think:
```
cy.shadowGet('todo-list-item')
  .shadowFind('something')
  .shadowFind('something-else')
  .shadowClick();
```
But maybe `within` is a good thing to bring support as well.
